tests/helpers/data_generator.py

```diff
@@ -599,3 +603,12 @@ def create_database_genomic_incident(self, **kwargs):
 
     def _genomic_incident(self, **kwargs):
         return GenomicIncident(**kwargs)
+
+    def create_database_genomic_gc_validation_metrics(self, **kwargs):
+        incident = self._genomic_validation_metrics(**kwargs)
+        self._commit_to_database(incident)
+        return incident
```

Comment:
Does the variable name `incident` make sense here?

Reply:
bad copy/paste

Reply:
https://github.com/all-of-us/raw-data-repository/pull/2446/commits/efde063403420be04bf842379b75a019c3c010c4